src/esn.calendar.libs/app/services/event-duplicate.service.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+angular.module('esn.calendar.libs')
+  .factory('calEventDuplicateService', eventDuplicateService);
```

Comment:
We need to test this in production mode. You're mixing `factory`, `calEventDuplicateService`, `eventDuplicateService`, so I think it's highly likely that `ng-annotate` won't understand this.

Reply:
Perhaps there won't be an error even if `ng-annotate` doesn't understand this since the service/factory does not use any other services. But we still need to fix this IMO.